#////////////////////////////////////////////////////////////////////
#////////////////////////////////////////////////////////////////////
# script: restoreFiles.py
# author: Lincoln Harris
# date: 6.29
#
# Wondering if I can use s3_util library to restore glaciered files 
# on seqbot
#////////////////////////////////////////////////////////////////////
#////////////////////////////////////////////////////////////////////
import s3_util

global myBucket, myPrefix
myBucket = 'czbiohub-seqbot'
#myPrefix = 'fastqs/171120_A00111_0085_AH57YYDMXX/rawdata'
myPrefix = 'fastqs/171120_A00111_0086_BH55NVDMXX/rawdata'

fList = []
for queryDir in s3_util.get_files(bucket=myBucket, prefix=myPrefix):
	fList.append(queryDir)
	
#s3_util.restore_files(fList)
s3_util.get_status(fList)
#////////////////////////////////////////////////////////////////////
#////////////////////////////////////////////////////////////////////